src/components/components/ebay-tooltip-base/component-browser.js

```diff
@@ -10,10 +10,29 @@ module.exports = {
         this.emit('base-collapse');
     },
 
+    handleKeydown(e) {
+        if (e.key === 'Escape') {
+            const {
+                input: { type },
+            } = this;
+            const container = this.getEl('container');
+            const hostClass = `${type}__host`;
+            const hostSelector = `.${hostClass}`;
+            const host = container.querySelector(hostSelector);
+            if (host) {
+                this.collapse();
+            }
+        }
+    },
+
     onMount() {
         if (this.input.type !== 'dialog--mini') {
             this._setupMakeup();
         }
+
+        this.subscribeTo(window).on('keydown', (e) => {
```

Comment:
You can/should probably use this instead: https://github.com/eBay/ebayui-core/blob/master/src/common/event-utils/index.js#L32

Example usage here: https://github.com/eBay/ebayui-core/blob/master/src/components/components/ebay-dialog-base/component.js#L37

Reply:
I think the event delegate should be `this.rootEl` instead of `window`.

Reply:
So I tried that and it breaks the tooltip... the tooltip doesn't show up at all.